src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -381,6 +386,9 @@ export class InputDatePicker implements LabelableComponent, FormComponent, Inter
   }
 
   disconnectedCallback(): void {
+    if (this.containerDiv) {
+      this.containerDiv.removeEventListener("transitionstart", this.transitionStartHandler);
```

Comment:
nitpick: Will this work? `this.containerDiv?.removeEventListener("transitionstart", this.transitionStartHandler);`